data/json/furniture.json

```diff
@@ -1349,7 +1349,7 @@
     "id": "f_bigmirror",
     "name": "standing mirror",
     "symbol": "{",
-    "description": "Lookin' good- is that blood?",
+    "description": "Lookin' sharp!  Wait, is that blood?",
```

Comment:
This is one of those lateral changes just for the sake of change.

Reply:
Changed due to that hyphen looking weird. Should at least be a space on either side. I'll do that, then, instead.